### PR TITLE
[DOCFIX] Remove deprecated sections from journal docs

### PR DESCRIPTION
**Pull request by Xenorith**
also update `init format` command to be able to format journal on all masters but avoid formatting workers

### CODE REVIEW

**Comment by Xenorith** [comment]
i removed sections based on the fact that the CLI commands to execute the operations no longer exist. if the sections are incorrectly removed, then the corresponding CLI command needs to be readded

File: docs/en/operation/Journal.md [hunk 1]
```diff
@@ -336,44 +160,6 @@ make the checkpointing a few times faster(depending how many threads are used).
...
-can help avoid leading master journal logs from growing unbounded when Alluxio is running in HA mode.
-
-Starting from version 2.4, Alluxio embedded journal HA mode supports automatically transferring checkpoints from standby masters to the leading master. 
-The leading master can use those checkpoints as taken locally to truncate its journal size without causing temporary service unavailability. 
-No need to manually transfer leadership anymore.
-
-If HA mode is not an option, the following command can be used to manually trigger the checkpoint:
-
-```shell
-$ ./bin/alluxio fsadmin journal checkpoint
```
Comment (jja725) [inline docs/en/operation/Journal.md:356]: I notice that we are not doing checkpointing in leading master... Which means the journal would be growing slowly if there's only one master and eventually cause problem. Is it true? @yuzhu @Xenorith @beinan

Comment (Xenorith) [reply docs/en/operation/Journal.md:356]: i don't have an answer to this question, but i also don't think it's blocking this PR. if determined that the command is needed, we'd need to add it back outside of this PR

**Comment by Xenorith** [comment]
@jja725 updated the docs after https://github.com/Alluxio/alluxio/pull/18174 PTAL again

**Review by jja725 — APPROVED** [review]
Hope we didn't go to the situation where we have to administer masters...

**Comment by jja725** [comment]
alluxio-bot, merge this please

**Comment by jja725** [comment]
alluxio-bot, cherry-pick this to branch-3-0.0.x please

**Pull request merged**